apps/api/src/app/shared/services/cache/cache.service.ts

```diff
@@ -98,7 +99,14 @@ export class CacheService implements ICacheService {
   private updateTtl(key: string, options?: CachingConfig) {
     const seconds = options?.ttl || this.cacheTtl;
 
-    return this.client.expire(key, seconds);
+    return this.client.expire(key, this.ttlVariant(seconds));
+  }
+
+  private ttlVariant(num) {
+    const randomNum = Math.random();
+    const variant = this.TTL_VARIANT_PERCENTAGE * randomNum;
```

Comment:
The variant should be,
`
const variant = this.TTL_VARIANT_PERCENTAGE * num;
`
The current equation will make the variance smaller by a random number.